src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -125,18 +124,17 @@ def post(self, request: Request) -> Response:
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
+        event.update({"data": event.pop("data", {})})
```

Comment:
change this to
`event.update(event.pop("data"), {})`
right?

we don't want to re-insert it with the same key, we want to pull out the data into the top level

Reply:
Ahh, nice catch. I assumed the request format was whats expected in the kafka payload.

Reply:
Yeah basically I wanted a way to sneak in unserialized data through the endpoint for now because it's not in use yet and it would be a big lift to keep everything in sync as we added new data as needed. we'll want to have everything specced out if/when the endpoint goes through the critical path